### PR TITLE
Type.resolveClass(); external classes cannot be resolved

### PR DESCRIPTION
**Pull request by GrafMine**

### CODE REVIEW

File: std/Type.hx [hunk 1]
```diff
@@ -116,6 +116,8 @@ extern class Type {
 	/**
 		Resolves an enum by name.
 
+		"On some targets external classes cannot be resolved"
```
Comment (Gama11) [inline std/Type.hx]: Why is this in quotation marks?

Comment (GrafMine) [inline std/Type.hx]: fixed

**Comment by Simn** [comment]
This is very unspecific and I don't think it actually adds anything. It's basically saying "This function may or not work".

For starters, why is this on resolve**Enum**? Then it raises the question what *external* even means. I assume this is about native types?

I think the Type API is only supposed to work with Haxe-defined types in general, so I guess our documentation should focus on that.

**Comment by RealyUniqueName** [comment]
Looks like this change was intended for `Type.resolveClass()` which indeed cannot resolve native types on JS which is also confusing for newcomers :)

**Comment by GrafMine** [comment]
`For starters, why is this on resolveEnum? `
fixed sorry)

**Comment by Simn** [comment]
I'll go ahead and close this for reasons previous stated.

**Pull request closed without merging**